components/clarity-repl/src/repl/session.rs

```diff
@@ -248,8 +248,15 @@ impl Session {
             cmd if cmd.starts_with("::get_contracts") => {
                 self.get_contracts().unwrap_or("No contract found".into())
             }
+            cmd if cmd.starts_with("::get_burn_block_height") => self.get_burn_block_height(),
+            cmd if cmd.starts_with("::get_stacks_block_height") => self.get_block_height(),
             cmd if cmd.starts_with("::get_block_height") => self.get_block_height(),
-            cmd if cmd.starts_with("::advance_chain_tip") => self.parse_and_advance_chain_tip(cmd),
```

Comment:
Even though it's removed from the docs, we want to keep it available (avoid breaking changes)